src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -229,6 +229,11 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
 
                 project.RawRuntimeOptions = runtimeOptions.ToString();
             }
+            else if (File.Exists(Path.Combine(projectPath, $"{projectName}.runtimeconfig.json")))
```

Comment:
@sokket Can you comment  on this intention? If this is indeed the intention I'll go ahead and add a test.